src/main/java/frc/robot/subsystems/vision/NoteCamera.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems.vision;
+
+
+import org.littletonrobotics.junction.Logger;
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+public class NoteCamera {
+
+  PhotonCamera camera;
+  String path;
+
+  private final double MIN_AREA = 10;
+
+  public NoteCamera(String name) {
+    // This is a constructor
+    camera = new PhotonCamera(name);
+    path = "Vision/" + name.replace("_", "");
```

Comment:
place into constant, if used elsewhere, make global and move to common folder

Reply:
the path name? (also you made this comment twice btw) probably but its used exactly once at the top of 3 files in vision/ and nowhere else. if such a change was made it should probably be standardized and constants as a whole should be reorganized